derex/runner/cli.py

```diff
@@ -99,14 +99,51 @@ def reset_mysql_cmd(project):
     from derex.runner.docker import wait_for_mysql
 
     if not check_services(["mysql"]):
-        click.echo("Mysql service not found.\nMaybe you forgot to run\nddc-services up -d")
+        click.echo(
+            "Mysql service not found.\nMaybe you forgot to run\nddc-services up -d"
+        )
         return
     wait_for_mysql()
     execute_mysql_query(f"CREATE DATABASE IF NOT EXISTS {project.mysql_db_name}")
     reset_mysql(project)
     return 0
 
 
+@derex.command(name="provision-forum")
+@click.pass_obj
+@ensure_project
+def provision_forum_cmd(project):
+    """Prime the elasticsearch index for the forum service"""
+    from derex.runner.docker import check_services
+    from derex.runner.compose_utils import run_compose
+
+    if "derex.forum" not in project.config.get("plugins", {}):
+        click.echo(
+            "Forum is not enabled for this project.\n"
+            "Enable it by installing the derex.forum plugin."
+        )
+        return
+
+    if not check_services(["elasticsearch"]):
+        click.echo(
+            "Elasticsearch service not found.\nMaybe you forgot to run\nddc-services up -d"
+        )
+        return
+
+    args = [
+        "exec",
```

Comment:
A `run`, `--rm` would be better: you might want to prime your indexes before booting up the forum.